### PR TITLE
Create fire stations map blog post

### PR DESCRIPTION
**Pull request by gregbunce**
WIP.  just checking how everything lines up, etc.

### CODE REVIEW

File: _posts/2022-04-14-nearest-fire-stations-map.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  
+### The SGID
+{: .text-left}
+  
+In Utah, the best place to begin any GIS project is with the [State Geographic Information Datasource]({% link data/index.html %}#welcome-to-the-state-geographic-information-database-sgid) (SGID). The SGID is hosted by the [Utah Geospatial Resource Center](https://gis.utah.gov/) (UGRC) and contains hundreds of statewide geospatial datasets, many of which are considered framework data that are used in larger government systems such as 911, voting, and taxation. Because of this, data in the SGID is updated frequently and adheres to established [policies and standards]({% link about/policy/sgid/index.md %}#internal-database-standards).
```
Comment (steveoh) [inline _posts/2022-04-14-nearest-fire-stations-map.md:21]: you can use link index.html here instead of gis.utah.gov

**Review by steveoh — COMMENTED** [review]
I really like these bite sized applied GIS posts. Nice work mr bunce.

File: _posts/2022-04-14-nearest-fire-stations-map.md (hunk 1 above)
Comment (gregbunce) [reply _posts/2022-04-14-nearest-fire-stations-map.md:21]: makes sense.  thx

**Comment by gregbunce** [comment]
"bite sized" was the goal.

**Review by steveoh — APPROVED** [review]

**Pull request merged**